README.md

```diff
@@ -1 +1,43 @@
-# Hakuna-Matata
\ No newline at end of file
+# Hakuna-Matata :lion_face: 
+Live link : https://gsg-g10.github.io/Hakuna-Matata/.
+
+Animal search page for animal lovers.
+
+## Team members :heart: 
+ * Nizar Zakout .
+ * Raghad Mezied . 
+ * Ahmad Omar.
+ * Nadia Tamimi.
+
+ ## Project Description :page_facing_up: 
+ Home Page displays a input search bar to write the name of the animal you are searching for , If you write the the first letter of the animal name a list of animal with the the same letter you write will aprear and then the list will become shorter when you input other letters. 
+
+There is also an audio button you can play while you are searching for the animal to enjoy your time on our page .
+
+## User Stories: :astonished: 
+User search about animals images.
+
+ * I want to be able to search about the animal i want.
+ * I want to be able to see all the options name of animal with same letters.
+
+
+## User Journey :world_map: 
+
+In your journey at our page you will find a nice logo of (simba, pumbaa and timon) , an audio to enjoy your trip and a input search bar to search the animal whose picture you want to see .
+
+While you are writing the name of the animal a small list will apeare under the input search bar with all animal that have for name the first letter you write , If you put more letter  shorter the list of animal will become and closer you will be from the animal you want . And don't be afraid that there is no search bar you have to cilck enter or the name that apear in the search .
+
+HAVE A NICE JOURNEY :hearts: 
+
+## Installing the project locally: :inbox_tray: 
+
+- git clone [Hakuna Matata](https://github.com/GSG-G10/Hakuna-Matata.git)
+- cd AnimeChan
+- go live server
```

Comment:
### Write these lines:
```
## Running the project locally: :inbox_tray:
- git clone https://github.com/GSG-G10/Hakuna-Matata.git
- cd Hakuna-Matata
- npm i
- npm start
```
#### instead of:
```
## Installing the project locally: :inbox_tray:
- git clone [Hakuna Matata](https://github.com/GSG-G10/Hakuna-Matata.git)
- cd AnimeChan
- go live server
```
I think it's better :)